scripts/ci/debian/control

```diff
@@ -0,0 +1,15 @@
+Source: mono-llvm
+Priority: optional
+Maintainer: Alex Kyte <alkyte@microsoft.com>
+Build-Depends: debhelper (>= 8.0.0)
```

Comment:
This line is where you specify which packages are required to build - for example, the upstream LLVM 3.9 package's build depends are here: http://sources.debian.net/src/llvm-toolchain-snapshot/1:3.9~svn274438-1/debian/control/#L6-L11

If you specify this correctly, you can generate a "good" source package which can be passed through to the chroot package builder, which makes building on multiple architectures WAY easier.